windows/src/developer/TIKE/xml/help/contexthelp.xml

```diff
@@ -226,14 +201,51 @@
     </Control>
   </Form>
 
+  <Form Name="context/keyboard-editor#toc-on-screen-tab">
+    <Control Name="chkVKDisplayUnderlyingChars" Title="Display Underlying Character Layouts">
+      <p>If selected, then the visual keyboard will only be used if the current underlying layout is active. </p>
+    </Control>
+    <Control Name="chkVKInclude102key" Title="">
```

Comment:
Could add a title like "102nd Key"

Reply:
There's a whole lotta tidyup we can do in contexthelp.xml. I'll leave this for now because much of it is documented in #2131 (There is some old content there to be removed as well).